src/components/calculator/Calculator.jsx

```diff
@@ -0,0 +1,45 @@
+import React, {useRef, useEffect} from 'react';
+import './calculator.css';
+import { btns, BTN_ACTIONS } from './btnConfig';
+
+const Calculator = () => {
+
+    const btnsRef = useRef(null)
+    const expRef = useRef(null)
+
+    useEffect(()=>{
+        const btns = Array.from(btnsRef.current.querySelectorAll('button'));
+
+        btns.forEach(e => e.style.height = e.offsetwidth + 'px')
+    },[])
+
+    const btnClick = (item) => {
+        console.log(item)
+    }
+
+    return (
+      <div className='calculator'>
+        <div className="calculator__result">
+            <div ref={expRef} className="calculator__result__exp">
+                999999
+            </div>
+            <div className="calculator__result__exp"></div>
+        </div>
+        <div ref={btnsRef} className="calculator__btns">
+            {
+                btns.map((item, index)=>(
+                    <button 
+                        key={index} 
+                        className={item.class}
+                        onClick={() => btnClick(item)}
```

Comment:
- [ ] Kindly make sure to display the calculator correctly. Currently, your calculator is not displayed.

https://user-images.githubusercontent.com/70269507/176377030-3063add3-91ca-435e-8137-dab418f90410.mov

Reply:
can you provide more feedback or error, because from my side everything is working correctly. I don't know where to start checking

Reply:
done